### PR TITLE
Operation timers should be removed when operation completes.

### PR DESCRIPTION
**Pull request by tsegismont**
Otherwise, the application might throw an OOM.
This would happen with a big enough circuit breaker timeout and a result value that retains a relatively large amount of memory.

Issue was reported by @pierDipi as a comment to #58 but it seems different than the original topic.

### CODE REVIEW

**Comment by tsegismont** [comment]
@pierDipi mind taking a look?

File: src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java [hunk 1]
```diff
@@ -16,16 +16,8 @@
...
-import io.vertx.circuitbreaker.CircuitBreakerState;
-import io.vertx.circuitbreaker.OpenCircuitException;
-import io.vertx.circuitbreaker.TimeoutException;
-import io.vertx.core.Context;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.Promise;
-import io.vertx.core.Vertx;
+import io.vertx.circuitbreaker.*;
+import io.vertx.core.*;
```
Comment (pierDipi) [inline src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java:20]: guessing conventionally the Vert.x project conventions allow star imports

**Review by pierDipi — APPROVED** [review]
LGTM

Thanks!

File: src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java (hunk 1 above)
Comment (tsegismont) [reply src/main/java/io/vertx/circuitbreaker/impl/CircuitBreakerImpl.java:20]: yes

**Pull request merged**